### PR TITLE
Features/expose retweets with factorized parameters

### PR DESCRIPTION
**Pull request by rodrigoaguilera**

### CODE REVIEW

File: tweetledee/userrss_nocache.php [hunk 1]
```diff
@@ -112,7 +112,7 @@
 header("Content-Type: application/rss+xml");
 header("Content-type: text/xml; charset=utf-8");
 
-$renderer = new RssRenderer();
+$renderer->using_client($client);
```
Comment (rodrigoaguilera) [inline tweetledee/userrss_nocache.php]: Isn't the renderer missing here @Riduidel ?

Comment (Riduidel) [reply tweetledee/userrss_nocache.php]: yes ideed, and it is even present in [#78](https://github.com/tweetledee/tweetledee/pull/78/files#diff-0613fc4cd74ffe369eeb2bfdeec69290R123)

**Comment by rodrigoaguilera** [comment]
I am missing some "recursion_limit" for when load_parameters() is called.

I will review more thoroughly and try to run it in a test environment.

**Comment by rodrigoaguilera** [comment]
I have added the bits that I failed to include during the rebase and tested locally both the inclusion of retweets and the multiple images display.

@Riduidel  The master branch should be now equivalent to your live branch

Thank you for this great feature!

I am going to open an issue about completing the documentation with the `rl` parameter.

I think I am going to roll a new release since we are not even in alpha phase. bugs can happen.

**Pull request merged**